entrypoint.sh

```diff
@@ -54,4 +55,12 @@ if check_duplicate_msg == "true"
   end
 end
 
+ if delete_prev_regex_msg != nil
+  coms.each do |n|
```

Comment:
I believe it's possible for `coms` to be `nil` here (if `check_duplicate_msg` is `false`), making the script error out. Can someone have a look?

Reply:
Yeah. I'll fix it.

Reply:
Not sure if this the root but we are seeing an issue when we pass msg="something", check_for_duplicate_msg="false" and delete_prev_regex_msg="nil":

> /entrypoint.sh:60:in `<main>': undefined method `each' for nil:NilClass (NoMethodError)

Reply:
@genevieve Oh; because those get passed in as strings - `"false"`, and `"nil"`, not actually `false` and `nil`. Which means that they're truthy.

Reply:
Ah it's because we use the image in a github action so even though we pass `false` or nothing, it will quote them before passing them as arguments to the docker command.

Reply:
Actually, this is happening for us regardless of whether we rely on the defaults or not because of the way that the github action passes the values to the docker command. We weren't explicitly setting `delete_prev_regex_msg` and it would pass in `"nil"`.